### PR TITLE
Removed mocha from Active Record Part 2

### PR DESCRIPTION
**Pull request by ronakjangir47**
r? @kaspth

### CODE REVIEW

File: activerecord/test/cases/migration/command_recorder_test.rb [hunk 1]
```diff
@@ -31,7 +31,8 @@ def create_table(name); end
       end
 
       def test_unknown_commands_delegate
-        recorder = CommandRecorder.new(stub(:foo => 'bar'))
+        recorderStruct = Struct.new(:foo)
```
Comment (kaspth) [inline activerecord/test/cases/migration/command_recorder_test.rb]: No camel cased variables please. This variable is better inlined too.

File: activerecord/test/cases/explain_test.rb [hunk 2]
```diff
@@ -39,38 +39,49 @@ def test_exec_explain_with_no_binds
...
+        expected = sqls.map {|sql| "EXPLAIN for: #{sql}\nquery plan #{sql}"}.join("\n")
+        assert_equal expected, base.exec_explain(queries)
+      end
     end
 
     def test_exec_explain_with_binds
-      cols = [Object.new, Object.new]
-      cols[0].expects(:name).returns('wadus')
-      cols[1].expects(:name).returns('chaflan')
+      objectStruct = Struct.new(:name)
```
Comment (kaspth) [inline activerecord/test/cases/explain_test.rb]: Just go with `object`.

File: activerecord/test/cases/fixtures_test.rb [hunk 3]
```diff
@@ -408,9 +408,11 @@ def test_accessor_methods_with_multiple_args
   end
 
   def test_reloading_fixtures_through_accessor_methods
+    topicStruct = Struct.new(:title)
```
Comment (kaspth) [inline activerecord/test/cases/fixtures_test.rb]: Use `topic`.

**Comment by ronakjangir47** [comment]
@kaspth Thanks for the review. Updated PR accordingly

**Comment by kaspth** [comment]
Thanks!

**Pull request merged**